ui/app/AppLayouts/Profile/popups/SyncDeviceCustomizationPopup.qml

```diff
@@ -19,8 +19,8 @@ import "../stores"
 StatusDialog {
     id: root
 
-    property DevicesStore devicesStore
-    property AdvancedStore advancedStore
+    property var devicesStore
```

Comment:
Because it can't be mocked, ie in storybook:
```qml
SyncingView {
    advancedStore: QtObject {
        readonly property bool isDebugEnabled: ctrlDebugEnabled.checked
    }
    devicesStore: QtObject {
        function generateConnectionStringAndRunSetupSyncingPopup() {
            logs.logEvent("devicesStore::generateConnectionStringAndRunSetupSyncingPopup()")
        }
    ...
}
```